cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/analyzer/ReplicationFactorChangeTest.java

```diff
@@ -51,7 +54,7 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
-import static com.linkedin.kafka.cruisecontrol.analyzer.AnalyzerUnitTestUtils.goal;
+import static com.linkedin.kafka.cruisecontrol.analyzer.AnalyzerUnitTestUtils.*;
```

Comment:
Nit: Can we avoid static import?

Reply:
You mean removing the wild card import right?